apsuite/orbcorr/orbit_correction.py

```diff
@@ -71,6 +70,9 @@ def __init__(self, model, acc, params=None, corr_system='SOFB'):
             pass
         else:
             raise ValueError('Corretion system must be "SOFB" or "FOFB"')
+        if self.params.enblrf and model.cavity_on is False:
+            raise Exception("It is necessary to turn the cavity on if it is"
+                            " to be used in correction..")
```

Comment:
I think `Exception` is too general. What do you think of the suggestion below?
```suggestion
            raise ValueError(
                "It is necessary to turn on the cavity if it is "
                "to be used in correction."
            )
```